### PR TITLE
Fix and cleanup functional tests with ccd

### PR DESCRIPTION
**Pull request by sabahirfan**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/cmc-claim-store/values.preview.template.yaml [hunk 1]
```diff
@@ -46,8 +46,11 @@ java:
     CMC_DB_USERNAME: cmc
     CMC_DB_PASSWORD: cmc
     CORE_CASE_DATA_API_URL: http://${SERVICE_NAME}-data-store-api
-    FEATURE_TOGGLES_CCD_ENABLED: false
-    FEATURE_TOGGLES_CCD_ASYNC_ENABLED: true
+    FEATURE_TOGGLES_CCD_ENABLED: true
+    FEATURE_TOGGLES_CCD_ASYNC_ENABLED: false
```
Comment (ckalista) [inline charts/cmc-claim-store/values.preview.template.yaml]: As discussed, we are OK with that until we find a fix that allows us to turn that back on while also having CCD on. At least with this PR we can start exercising most tests agains CCD. Nearly there - good job 👍

File: src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/PostClaimOrchestrationHandler.java [hunk 2]
```diff
@@ -17,37 +17,21 @@
...
-        indicators.getSealedClaimUpload().equals(YesNoOption.NO);
-    private final Predicate<ClaimSubmissionOperationIndicators> isUploadClaimReceiptSuccess = indicators ->
-        indicators.getClaimIssueReceiptUpload().equals(YesNoOption.NO);
-    private final Predicate<ClaimSubmissionOperationIndicators> isRpaOperationSuccess = indicators ->
-        indicators.getRpa().equals(YesNoOption.NO);
-    private final Predicate<ClaimSubmissionOperationIndicators> isNotifyStaffSuccess = indicators ->
-        indicators.getStaffNotification().equals(YesNoOption.NO);
-    private final Predicate<ClaimSubmissionOperationIndicators> isNotifyCitizenSuccess = indicators ->
-        indicators.getClaimantNotification().equals(YesNoOption.NO);
-
```
Comment (sabahirfan) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/events/claim/PostClaimOrchestrationHandler.java]: NOTE: Clean up all these predicate functions as these were never used as the assignment target for a lambda expression or method reference. Rather these were misused to muddle the code. The names were also misleading for these.

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java [hunk 3]
```diff
@@ -82,19 +81,6 @@
     private final JobSchedulerService jobSchedulerService;
     private final CCDCreateCaseService ccdCreateCaseService;
 
-    private static BiFunction<ClaimSubmissionOperationIndicators, ClaimDocumentType, ClaimSubmissionOperationIndicators>
```
Comment (sabahirfan) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java]: NOTE: Cleaned up this BiFunction as it was never used as the assignment target for a lambda expression or method reference. Rather it was misused to muddle the code.

File: charts/cmc-claim-store/values.preview.template.yaml (hunk 1 above)
Comment (sabahirfan) [reply charts/cmc-claim-store/values.preview.template.yaml]: 👍

**Pull request merged**